interface/usergroup/user_admin.php

```diff
@@ -315,7 +315,7 @@ function authorized_clicked() {
 <td><span class=text><?php echo xlt('Last Name'); ?>: </span></td><td><input type="text" name=lname id=lname style="width:150px;"  class="form-control" value="<?php echo attr($iter["lname"]); ?>"><span class="mandatory"></span></td>
 <td><span class=text><?php echo xlt('Default Facility'); ?>: </span></td><td><select name=facility_id style="width:150px;" class="form-control">
 <?php
-$fres = $facilityService->getAllBillingLocations();
+$fres = $facilityService->getAllServiceLocations();
```

Comment:
I just switched the method here but I would like to know if we need a new method to list both billing and service locations or if using `getAllServiceLocations` is enough.

Reply:
Good insight. I thought about this when I did fix and decided if it's only a billing location there'd be no reason for user to be defaulted to just a billing location.

Reply:
good question and good point, think we're safe with `ServiceLocations` for now :)